uid-pages/form-case-overview/test/specs/formCaseOverview.feature

```diff
@@ -22,4 +22,86 @@ Feature: Form case overview
     And The open case business data is defined
     When I visit the open case index page
     Then The correct BDM headers are visible
-    And The incorrect BDM headers don't exist
\ No newline at end of file
+    And The incorrect BDM headers don't exist
+
```

Comment:
Maybe should remove the "i don't see" assertions

Reply:
:heavy_check_mark: